packages/driver/src/cypress/error_messages.js

```diff
@@ -0,0 +1,1795 @@
+const _ = require('lodash')
+const { stripIndent } = require('common-tags')
+const capitalize = require('underscore.string/capitalize')
+
+const divider = (num, char) => {
+  return Array(num).join(char)
+}
+
+const format = (data) => {
+  if (_.isString(data)) {
+    return _.truncate(data, { length: 2000 })
+  }
+
+  if (_.isObject(data)) {
+    return JSON.stringify(data, null, 2)
+  }
+
+  return data
+}
+
+const formatConfigFile = (configFile) => {
+  if (configFile === false) {
+    return '`cypress.json` (currently disabled by --config-file=false)'
+  }
+
+  return `\`${format(configFile)}\``
+}
+
+const formatRedirects = (redirects = [], listIndentSize) => {
+  return _.map(redirects, (redirect) => {
+    return `${' '.repeat(listIndentSize)}  - ${redirect}`
+  })
+}
+
+const formatProp = (memo, field) => {
+  const { key, value } = field
+
+  if (value != null) {
+    memo.push(`${capitalize(key)}: ${format(value)}`)
+  }
+
+  return memo
+}
+
+const cmd = (command, args = '') => {
+  const prefix = command.startsWith('Cypress') ? '' : 'cy.'
+
+  return `\`${prefix}${command}(${args})\``
+}
+
+const getScreenshotDocsPath = (cmd) => {
+  if (cmd === 'Cypress.Screenshot.defaults') {
+    return 'screenshot-api'
+  }
+
+  return 'screenshot'
+}
+
+const getRedirects = (obj, phrase, listIndentSize) => {
+  const redirects = obj.redirects ?? []
+
+  if (!redirects.length) {
+    return ''
+  }
+
+  const word = redirects.length > 1 ? 'times' : 'time'
+
+  const list = formatRedirects(redirects, listIndentSize)
+
+  return [
+    `${phrase} '${redirects.length}' ${word} to:`,
+    '',
+    `${list.join('\n')}`,
+  ].join('\n')
+}
+
+const getHttpProps = (fields = []) => {
+  return _
+  .chain(fields)
+  .reduce(formatProp, [])
+  .join('\n')
+  .value()
+}
+
+const cyStripIndent = (str, indentSize) => {
```

Comment:
Can you add a comment here explaining why someone would need to use function this over the included `stripIndent` function?